jck/compiler.lang/playlist.xml

```diff
@@ -56,18 +60,22 @@
 	</test>
 	<test>
 		<testCaseName>jck-compiler-lang-CLSS-Group2</testCaseName>
-		<disabled>
-			<comment>Disabled on aix due to RTC: 144784</comment>
-			<platform>ppc64_aix(_mixed)?</platform>
-			<version>11</version>
-			<impl>ibm</impl>
-		</disabled>
-		<disabled>
-			<comment>Disabled on aix due to RTC: 144784</comment>
-			<platform>ppc64_aix(_mixed)?</platform>
-			<version>11</version>
-			<impl>openj9</impl>
-		</disabled>
+		<disables>
+			<disable>
+				<comment>Disabled on aix due to RTC: 144784</comment>
+				<platform>ppc64_aix(_mixed)?</platform>
+				<version>11</version>
+				<impl>ibm</impl>
+			</disable>
+		</disables>
+		<disables>
+			<disable>
+				<comment>Disabled on aix due to RTC: 144784</comment>
+				<platform>ppc64_aix(_mixed)?</platform>
+				<version>11</version>
+				<impl>openj9</impl>
+			</disable>
+		</disables>
```

Comment:
`<disable>` should be all under one `<disables>`. Please see https://github.com/eclipse-openj9/openj9/blob/master/test/docs/OpenJ9TestUserGuide.md#exclude-test-against-multiple-criteria for more detail. Please check the rest of PR. Thanks.